samples/stock/demo/stock-design-black-and-white/demo.js

```diff
@@ -0,0 +1,191 @@
+(async () => {
+    const data = await fetch(
+        'https://www.highcharts.com/samples/data/new-intraday.json'
+    ).then(response => response.json());
+
+    Highcharts.setOptions({
+        lang: {
+            rangeSelectorZoom: ''
+        }
+    });
+
+    Highcharts.stockChart('container', {
+        chart: {
+            plotBorderWidth: 1,
+            plotBorderColor: '#000000',
+            marginRight: 30
+        },
+        title: {
+            text: ''
+        },
+        rangeSelector: {
+            buttons: [{
```

Comment:
Can we add a [title](https://api.highcharts.com/highstock/rangeSelector.buttons.title) to the buttons? So that the native tooltip and assistive tech will pick it up.

Reply:
@TorsteinHonsi Yes, of course! What should be the content of such title? Simple 'M15' or more descriptive like '15 minutes'?

Reply:
Got it, thank you